Source/WebCore/loader/cache/CachedFontLoadRequest.h

```diff
@@ -81,14 +81,14 @@ class CachedFontLoadRequest final : public FontLoadRequest, public CachedFontCli
 
     void fontLoaded(CachedFont& font) final
     {
-        ASSERT_UNUSED(font, &font == m_font.get());
-        if (m_fontLoadRequestClient)
-            m_fontLoadRequestClient->fontLoaded(*this);
-
         if (m_font->didRefuseToLoadCustomFont() && m_context) {
             auto message = makeString("[Lockdown Mode] This font has been blocked: ", m_font->url().string());
             m_context->addConsoleMessage(MessageSource::Security, MessageLevel::Info, message);
         }
+
+        ASSERT_UNUSED(font, &font == m_font.get());
```

Comment:
Can we move this back to the top? Always nice to have the parameter assertion first.